tests/cecil-tests/GetterExceptionTest.cs

```diff
@@ -0,0 +1,81 @@
+using System;
+using Mono.Cecil;
+using System.Collections.Generic;
+using NUnit.Framework;
+using System.Linq;
+using Mono.Cecil.Cil;
+using Xamarin.Utils;
+using Xamarin.Tests;
+
+namespace Cecil.Tests {
+	[TestFixture]
+	public class GetterExceptionTest {
+		readonly HashSet<string> exceptionsToSkip = new ()
+		{
+			"AudioQueueException",
+			"ModelNotImplementedException",
+			"You_Should_Not_Call_base_In_This_Method"
```

Comment:
Add a "," and the end, that way the next diff will be smaller. 
```suggestion
			"You_Should_Not_Call_base_In_This_Method",
```